### DISCUSSION TITLE
Hosting on FastVM

### DISCUSSION THREAD

**Discussion by ItsMacso** [jagrosh/MusicBot#979] [Troubleshooting]
Bot is setup like on windows, setted the startup command and after giving the task to playback a song the bot joins and than leaves in loop without a music playaback

**Comment by JulianGmp** [comment]
This sounds a bit similar to what I had a while ago, this could be an issue with the firewall blocking the UDP communication for the voice channel.

https://github.com/jagrosh/MusicBot/discussions/956
